examples/cymbal-bank/README.md

```diff
@@ -0,0 +1,129 @@
+# Cymbal Bank Examples
+
+This example shows how to deploy Cymbal Bank using the infrastructure created at [Enterprise Application blueprint](https://cloud.google.com/architecture/enterprise-application-blueprint).
+
+For more information about Cymbal Bank application, please visit [Bank of Anthos repository](https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/v0.6.4).
```

Comment:
Would it be simpler to link to the Bank of Anthos repo without a version?  Just thinking this link might get stale.

```suggestion
For more information about the Cymbal Bank application, please visit [Bank of Anthos repository](https://github.com/GoogleCloudPlatform/bank-of-anthos/blob/v0.6.4).
```